release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -78,6 +78,16 @@ Previously, secret information was used as the OAuth access token and OAuth auth
 If you are upgrading your cluster to {product-title} 4.6, old tokens from {product-title} 4.5 will still have the secret information exposed in the object name. By default, the expiration for tokens is 24 hours, but this setting can be changed by administrators. Sensitive data can still be exposed until all old tokens have either expired or have been deleted by an administrator.
 ====
 
+[id="ocp-4-6-nodes"]
+=== Nodes
```

Comment:
@codyhoag I added the Nodes section at the same location that it was in for 4.4. But that isn't necessarily the same order as the left nav. Let me know if you want me to move it so that it is down further.

Reply:
Yeah, go ahead and move it so it follows the left nav. I'll try and make sure we stay consistent with that has the 4.6 RNs expand over the next few weeks. Thanks!

Reply:
Thanks will do!